src/metabase/api/field.clj

```diff
@@ -78,6 +78,7 @@
       (create-field-values-if-needed! field))))
 
 
+;; TODO - not sure this is used anymore
```

Comment:
Yeah. That endpoint is still there, but I don't think it's used anymore by the frontend